src/dllapi.cpp

```diff
@@ -50,12 +50,18 @@ static void ClientCommand(edict_t *pEntity)
     META_RES res = MRES_IGNORED;
 
     std::string strCmd(CMD_ARGV(0));
+    
+    if (strCmd == "menuselect")
+    {
+        res = gSPGlobal->getMenuManagerCore()->ClientCommand(pEntity);
+        RETURN_META(res);
+    }
 
     const std::unique_ptr<CommandMngr> &cmdMngr = gSPGlobal->getCommandManagerCore();
-    std::shared_ptr<Player> player = gSPGlobal->getPlayerManagerCore()->getPlayerCore(pEntity);
+    std::shared_ptr<Player> player = gSPGlobal->getPlayerManagerCore()->getPlayerCore(spEdict);
     if (cmdMngr->getCommandsNum(ICommand::Type::Client))
     {
```

Comment:
Maybe move body condition to CmdManager->ClientCommand() like MenuManager?

Reply:
Fixed